src/index.html

```diff
@@ -11,12 +11,100 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Kanit:ital,wght@0,300;0,400;1,300;1,900&family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="img"
+        />
+      </a>
+      <nav class="navbar">
+        <ul class="list">
+          <li class="menu">
+            <a
+              href="#"
+              class="is-active link"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="menu">
+            <a
+              href="#"
+              class="link"
+            >
+              SAMSUNG
```

Comment:
Similar to the previous comment, the text 'SAMSUNG' should be transformed to uppercase using CSS rather than being written in uppercase directly in the HTML.